flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -266,23 +213,75 @@ var A320_Neo_LowerECAM_Engine;
                 }
             }
         }
-        setN1VibrationValue(_value, _force = false) {
-            if ((_value != this.N1VibrationValue) || _force) {
-                this.N1VibrationValue = _value;
+        setN1VibrationValue(value, force = false) {
+            if ((value != this.N1VibrationValue) || force) {
+                this.N1VibrationValue = value;
                 if (this.N1VibrationValueText != null) {
                     this.N1VibrationValueText.textContent = fastToFixed(this.N1VibrationValue, 1);
                 }
             }
         }
-        setN2VibrationValue(_value, _force = false) {
-            if ((_value != this.N2VibrationValue) || _force) {
-                this.N2VibrationValue = _value;
+        setN2VibrationValue(value, force = false) {
+            if ((value != this.N2VibrationValue) || force) {
+                this.N2VibrationValue = value;
                 if (this.N2VibrationValueText != null) {
                     this.N2VibrationValueText.textContent = fastToFixed(this.N2VibrationValue, 1);
                 }
             }
         }
     }
     A320_Neo_LowerECAM_Engine.EngineInfo = EngineInfo;
+    class EngineIgniter {
+        constructor(textElement) {
+            this.currentState = Definitions.IGN_STATE.NONE;
```

Comment:
**suggestion**: I believe `currentState` is no longer accessed outside of this type. Thus it should be prefixed with an `_`.

_Side-note: I'm not a fan of such naming patterns, however this seems to be the standard set in this project. It is the way it is 😀!_

Reply:
Done! 
_Noted :P _